.github/workflows/codecov-nightly.yml

```diff
@@ -11,6 +11,7 @@ jobs:
     runs-on: windows-latest
     env:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+      CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
I added this as a secret. No idea why this looks completely different than what we have for other packages. Or is this the way you do coverage now in R?

Reply:
the action is taken from the example here: 
https://github.com/r-lib/actions/blob/v2-branch/examples/test-coverage.yaml

Reply:
But why are we doing it like this and so complicated?? It's a one liner in appveyor

Reply:
And no, the GitHub pat is not the token. I think the token is required so that they know what repo to use no? I used one for all other codecov supported project and that's also how they explained the GitHub action integration

Reply:
Regarding the token: the docu of the GH-action says "no codecov token required for public repos". Don't ask me why :)
https://github.com/codecov/codecov-action#usage

Reply:
> But why are we doing it like this and so complicated?? It's a one liner in appveyor

In RE the idea was to execute code coverage nightly instead of on push/pr because it took so much time.
For the plots, running the coverage will be pretty fast (I assume), so we could integrate it into appveyor.yml instead of nightly.

Reply:
yeah but you can do what we do for ospsutie..etc an appveyor.nightly 
Anyhow...if this works :)